src/common/workspace-store.ts

```diff
@@ -44,23 +71,39 @@ export class Workspace implements WorkspaceModel, WorkspaceState {
     }
   }
 
+  /**
+   * Is workspace managed by an extension
+   */
   get isManaged(): boolean {
     return !!this.ownerRef;
   }
 
+  /**
+   * Get workspace state
+   *
+   */
   getState(): WorkspaceState {
     return {
       enabled: this.enabled
     };
   }
 
+  /**
+   * Push state
+   *
+   * @interal
```

Comment:
I really think that any API marked as `@internal` should be not exported in a similar style to #1546.

Reply:
I also think the same but we ran out of time. See #1435.

Reply:
Sounds good